code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,6 +319,20 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
+	if(method==INGEST && is_species(M, /datum/species/squid))
+		to_chat(M, "<span class='danger'>Your tongue shrivels as you taste the salt! It burns!</span>")
+		if(prob(25))
+			user.emote("scream")
+		M.adjustFireLoss(5, TRUE)
+		M.Sleeping(20, 0)
```

Comment:
What's with the sleeping?

Reply:
Another thought was hallucinate. But I felt damage would be thematic for something that makes them spasm violently

Reply:
Neither sleeping nor hallucinating makes sense here.

Reply:
Black powder supposedly makes plasmemes hallucinate. Are you against FUN

Reply:
In this case, yes.